netbox/templates/dcim/virtualdevicecontext.html

```diff
@@ -0,0 +1,76 @@
+{% extends 'generic/object.html' %}
+{% load helpers %}
+{% load plugins %}
+{% load render_table from django_tables2 %}
+
+{% block breadcrumbs %}
+  <li class="breadcrumb-item"><a href="{% url 'dcim:devicerole_list' %}">Device Roles</a></li>
+{% endblock %}
+
+{% block extra_controls %}
+  {% if perms.dcim.add_interface %}
+    <a href="{% url 'dcim:interface_add' %}?device={{ object.device.pk }}&vdc={{ object.pk }}" class="btn btn-sm btn-primary">
+      <span class="mdi mdi-plus-thick" aria-hidden="true"></span> Add Interface
+    </a>
+  {% endif %}
+{% endblock extra_controls %}
```

Comment:
This seems confusing, as interfaces are _added_ to devices and _assigned_ to VDCs. It seems unlikely that a user will want to create a new interface on a device while viewing a VDC. (Also the `vdc` parameter in the URL should be `vdcs`.)

Reply:
We can remove it for now, however adding Vlan interfaces could definitely be 1 way this could be useful.  We can re-add based on feedback.